Schemata/sarif-schema.json

```diff
@@ -1270,6 +1237,11 @@
           "$ref": "#/definitions/message"
         },
 
+        "analysisTarget": {
```

Comment:
>analysisTarget [](start = 9, length = 14)

FYI, moving items around in the schema actually has impact on the generated OM. The class definition code emit follows the schema. This ordering appears to have some influence on compilation/MSIL code gen (to use the C# case). Reflection adheres to this ordering. As a result, changes in schema ordering can result in changes to JSON data ordering as emitted by the C# object model. if we want to define a standard for ordering (such as everything is alphabetized at its property scope), we should do that and make a scrub.